src/client/actions/networkRequest.js

```diff
@@ -23,32 +28,35 @@ export const receiveStock = ({
   symbol
 });
 
-export const requestStock = symbol => ({
-  type: REQUEST_STOCK,
-  symbol
+export const setIsFetching = isFetching => ({
```

Comment:
Thinking through this more.  What about having `setFetching(stockSymbol)`, which sets `state.fetching` to the stock symbol, instead of `state.isFetching` to true (you'd delete state.isFetching)?  The advantage of this is that right now, there isn't anything that keeps track of the actual text being searched if the user starts typing in the search bar before the fetch is complete (as far as I can tell).

For marking a fetch operation as complete in the state, I see 2 options:
- use `setDoneFetching` as described above, and set `state.fetching = null`
- let `setSearchError` and `receiveStock` handle setting `state.fetching = null`, instead of having a dedicated action for this.  The reason I mention this option is that right now `setSearchError` and `receiveStock` shouldn't really be called without a corresponding `setIsFetching(false)` anyway.